### ISSUE TITLE
Command line switch to load with a file configuration

### ISSUE THREAD

**Issue by buffos** [bitfocus/companion#2408]
### Is this a feature relevant to companion itself, and not a module?

- [X] I believe this to be a feature for companion, not a module

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the feature

We have different projects and different layouts.
It would be nice to be able to load companion with a configuration file switch, that resets companion and loads that configuration.

I have search documentation and issues and have not found any mention of command line switches, hence this post

### Usecases

As part of automation. 
Personally for every project, I have a script that launches the required programs. Then I have to load the corresponding layout to companion. It is a step that can be avoided, especially when you have to setup everything for someone else to work.

**Comment by toomanybrians** [comment]
+1 I would like this as well.

My usecase is we have a Raspberry Pi "appliance" we use to program firmware for some of our devices, which we control via a Stream Deck w/ Companion. Works great!

We keep these devices up to date via a GitHub repo, which they sync down. It would be nice if we could configure Companion to use the configuration file directly from the repo location, instead of needing to have a script to copy the file out.

This would also be helpful when we are designing updates to the button and layout config, in that we could make updates which are directly written to the repo, rather than needing a script to copy the configuration file into the rep.

**Comment by joshperlman** [comment]
Agreed this would be great. We are looking for more ways to automate our show deployment. If we could use a script to load a companion file that would be an awesome feature to build on top of

**Comment by maverickmueller** [comment]
I was able to do this by swapping in the db file for our new deployed version in the appdata folder via batch script. Not ideal, but does the job

**Comment by rjmunro** [comment]
I think you can set an environment variable `COMPANION_CONFIG_BASEDIR` to set folder where you keep config, so you could create a folder for each different configuration and use this.

**Comment by DanTappan** [comment]
I was looking for a way to configure a new installation of companion using a configuration file from GitHub and I was redirected here as a duplicate. 

Has this feature actually been implemented? 

This would need to be something that works across releases, so copying the folder of database files and using the COMPANION_CONFIG_BASEDIR variable feels like a risky solution that might not be supported long term.
